README.md

```diff
@@ -587,7 +587,7 @@ This library uses [semantic versioning](http://semver.org/). For each release, t
 * Push tag to origin such as `git push origin x.x.x`.
 * Make a PR against `develop`
 * Wait for review. Once the PR is approved, merge it into `develop`
-* Fast-forward the master branch: `git checkout master && git merge --ff-only develop`
+* Fast-forward the master branch: `git checkout master && git merge --ff-only develop && git push`
```

Comment:
I think this should be `git push origin master`

Reply:
Fixed https://github.com/ably/ably-ios/pull/724/commits/625723b41a78113e4dd450ca92f4a702eb12f710